api-client/src/endpoints/hooks/useLoadFromEndpoint.ts

```diff
@@ -23,7 +23,7 @@ export const loadFromEndpoint = async <T>(
       setError(null)
     }
   } catch (e) {
-    setError(e)
+    setError(e instanceof Error ? e : null)
```

Comment:
I don't think we should ignore errors if it is not an instance of type Error. Maybe just use
```suggestion
    setError(e instanceof Error ? e : new Error())
```